### PR TITLE
fixing imports in .py files in `utils` and `tests`  

### LINKED ISSUE #179: Fix Import Paths for Python Files in Util Folder
The paths used for the imports in `GatorGrouper.py` as well as other python files like `group_random.py`, `group_rrobin.py`, `parse_arguments.py`, etc. are incorrect and produced errors in pylint` and TravisCI. These errors are related to the format of the `import` statements and prevent GatorGrouper from running correctly.

### PR DESCRIPTION
**Pull request by ilikerustoo**
# Description of Pull Request

@ilikerustoo  @paladp @yeej2 Fixed imports in the `.py` modules within `utils` and `test` so that `Gator Grouper` actually works while `pylint` and `pytest` do not throw any errors.

Fixes #179 :

## Type of Change

Please describe the pull request as one of the following: 

- [x] Bug fix
- [ ] Breaking change
- [ ] New feature
- [ ] Documentation update
- [x] Other (Please specify any helpful information below)

Refactored code, `pylint`, `pytest` fixes.

## Tags

Please tag those who are responsible for reviewing proposed changes below.

@Lancasterwu  @paladp @yeej2 @Michionlion @toccinAC

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -17,6 +16,7 @@ pytest-clarity = "*"
 autopep8 = "*"
 gspread =  "*"
 oauth2client = "*"
+pylint = "*"
```
Comment (Lancasterwu) [inline Pipfile]: Quick question here, does the order here matter at all?

Comment (ilikerustoo) [reply Pipfile]: Nope

**Comment by baldeosinghm** [comment]
Hey @toccinAC @paladp. Do you think we could get this merged asap?  I'm unable to work on my issue without this fix.

**Comment by aubreypc** [comment]
@Michionlion This is ready to merge!

**Pull request merged**